python/lsst/cp/pipe/defects.py

```diff
@@ -748,7 +768,8 @@ class FindDefectsTask(pipeBase.CmdLineTask):
     The task has two modes of operation, defect finding in raws and in
     master calibrations, which work as follows.
 
-    - Master calib defect finding
+    Master calib defect finding
+    ---------------------------
```

Comment:
The text below this point does not appear in the documentation. I'm pretty sure this should give you a prominent error from numpydoc, which is very intolerant of non-standard headers. Please [simulate subsections with bold text](https://developer.lsst.io/python/numpydoc.html#simulate-subsections-with-bold-text) instead.